electrumx/server/session.py

```diff
@@ -1291,6 +1291,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.client = 'RPC'
         self.connection.max_response_size = 0
+        self.framer.max_size = 0
```

Comment:
I'm not sure about this - I think we only want to have no limit in the electrumx_rpc client no?  I don't see a reason for unlimited incoming message size.

Reply:
Oh yes, you're right. For some reason I was thinking the limit is problematic both ways; but actually the framer is only used for incoming messages, so this line is not needed. I'll remove.